docs/lessons/lesson1-10/Boltdir/modules/package/tasks/init.rb

```diff
@@ -0,0 +1,71 @@
+#!/opt/puppetlabs/puppet/bin/ruby
```

Comment:
I think these just come packaged with Bolt - we shouldn't need to include them, and it doesn't look like they're used in the lab? Am I missing something?

Reply:
They were originally included as part of Lesson 6. Since they're packaged with Bolt I agree that it does seem redundant and can be removed.